src/components/Sidebar/InterestingPeople.js

```diff
@@ -19,7 +19,19 @@ const InterestingPeople = ({ users, onRefresh }) => (
         />
       </h4>
       <div className="InterestingPeople__divider" />
-      {users && users.map(user => <User key={user.name} user={user} />)}
+      {users &&
+        users.map(user => (
+          <User
+            key={user.name}
+            user={user}
+            usernameStyles={{
+              width: '120px',
```

Comment:
I'm not sure if we should have width hardcoded and if it's the best way to create usernameStyles props. Can't we fix the style inside <User> component directly?

Reply:
So User component is used in different places, the only way to add an ellipses via css is having a width on it for different cases.